### ISSUE TITLE
wishlist item - task grouping.

### ISSUE THREAD

**Issue by kaushiknfs** [QTodoTxt/QTodoTxt2#60]
As with sort, grouping by the same would be nice.

**Comment by oroulet** [comment]
Can you explain what you mean?

**Comment by kaushiknfs** [comment]
When i sort todos by a param, lets say there should be option to group by sort.
If we Group by Sort on Project
+ProjectA
      item 1
      item 2
+ProjectB
      item3
      item4

**Comment by fuchsmich** [comment]
I think, grouping is not that easy to implement, because an item can belong to more than one group. But of course it would be a nice feature.

**Comment by oroulet** [comment]
Yes that is one thing I thought a lot about at the beginning. It is not impossible in qtodotxt2, it is purely visual, several graphics items can have links to the same task. Maybe worth a try..

**Comment by oroulet** [comment]
Need to make a new datamodel in python for tree...
